integration_tests/integration/analytics.spec.ts

```diff
@@ -435,112 +455,96 @@ context('Analytics section', () => {
+            action: 'opened',
+            label: 'MDI',
+          })
+        )
+      page
+        .getGraphFeedback('incentive-levels-by-age')
+        .click()
+        .then(() =>
+          gaSpy.shouldHaveSentEvent('incentives_event', {
+            category: 'Is this chart useful > Incentive level by age',
+            action: 'closed',
+            label: 'MDI',
+          })
+        )
+
+      page
+        .getGraphFeedback('entries-by-age')
+        .click()
+        .then(() =>
+          gaSpy.shouldHaveSentEvent('incentives_event', {
+            category: 'Is this chart useful > Behaviour entries by age',
+            action: 'opened',
+            label: 'MDI',
+          })
+        )
+      page
+        .getGraphFeedback('entries-by-age')
+        .click()
+        .then(() =>
+          gaSpy.shouldHaveSentEvent('incentives_event', {
+            category: 'Is this chart useful > Behaviour entries by age',
+            action: 'closed',
+            label: 'MDI',
+          })
+        )
     })
 
     it('users can submit feedback on charts', () => {
       testValidFeedbackSubmission(AnalyticsProtectedCharacteristics, [
-        ['incentivesByEthnicityFeedback', 'incentivesByEthnicityFeedbackForm'],
-        ['incentivesByAgeFeedback', 'incentivesByAgeFeedbackForm'],
-        ['incentivesByReligionFeedback', 'incentivesByReligionFeedbackForm'],
-        ['incentivesByDisabilityFeedback', 'incentivesByDisabilityFeedbackForm'],
-        ['incentivesBySexualOrientationFeedback', 'incentivesBySexualOrientationFeedbackForm'],
+        'population-by-age',
+        'incentive-levels-by-age',
+        'entries-by-age',
       ])
     })
 
     it('users will see errors if they submit invalid feedback on chart', () => {
       testInvalidFeedbackSubmission(AnalyticsProtectedCharacteristics, [
-        ['incentivesByEthnicityFeedback', 'incentivesByEthnicityFeedbackForm'],
-        ['incentivesByAgeFeedback', 'incentivesByAgeFeedbackForm'],
-        ['incentivesByReligionFeedback', 'incentivesByReligionFeedbackForm'],
-        ['incentivesByDisabilityFeedback', 'incentivesByDisabilityFeedbackForm'],
-        ['incentivesBySexualOrientationFeedback', 'incentivesBySexualOrientationFeedbackForm'],
+        'population-by-age',
+        'incentive-levels-by-age',
+        'entries-by-age',
       ])
     })
   })
 })
 
 function testValidFeedbackSubmission<PageClass extends AnalyticsPage>(pageClass: new () => PageClass, feedbackBoxes) {
-  feedbackBoxes.forEach(([feedbackBox, feedbackForm]) => {
+  feedbackBoxes.forEach(graphId => {
     let page = Page.verifyOnPage(pageClass)
 
     // open feedback box and select "yes"
-    page[feedbackBox].click()
-    page[feedbackForm].find('[name=chartUseful][value=yes]').click()
-    page[feedbackForm].submit()
+    page.getGraphFeedback(graphId).click()
+    page.getGraphFeedbackForm(graphId).find('[name=chartUseful][value=yes]').click()
```

Comment:
We refactored these 2 methods to (hopefully) simplify the calling code and reduce the number of accessor methods